server/config/toml.go

```diff
@@ -237,9 +237,27 @@ snapshot-interval = {{ .StateSync.SnapshotInterval }}
 snapshot-keep-recent = {{ .StateSync.SnapshotKeepRecent }}
 
 ###############################################################################
-###                         Store / State Streaming                         ###
+###                              State Streaming                            ###
 ###############################################################################
 
+# Streaming allows nodes to stream state to external systems
+[streaming]
+
+# streaming.abci specifies the configuration for the ABCI Listener streaming service
+[streaming.abci]
+
+# List of kv store keys to stream out via gRPC
+# Set to ["*"] to expose all keys.
+keys = [{{ range .Streaming.ABCI.Keys }}{{ printf "%q, " . }}{{end}}]
+
+# The plugin name used for streaming via gRPC
+# Supported plugins: abci_v1
+plugin = "{{ .Streaming.ABCI.Plugin }}"
+
+# stop-node-on-err specifies whether to stop the node on message delivery error
+stop-node-on-err = {{ .Streaming.ABCI.StopNodeOnErr }}
```

Comment:
How would one configure their node to not stream anything?

Reply:
It was suggested that I remove `enable` and use `plugin`. Leaving plugin blank will not enable anything. I need to add comments to that effect (ref: https://github.com/cosmos/cosmos-sdk/pull/13473#discussion_r1021924829).